packages/cli/src/commands/virtualdevices/events.ts

```diff
@@ -35,7 +35,7 @@ function buildTableOutput(tableGenerator: TableGenerator, data: EventInputOutput
 	return table.toString()
 }
 
-interface EventInputOutput {
+export type EventInputOutput = {
```

Comment:
Just want to double-check that this one wants to be exported, since it wasn't before.

Reply:
Oops, not intentional. Fixed.